src/lightning/pytorch/loggers/comet.py

```diff
@@ -241,21 +242,10 @@ def __init__(
         log.info(f"CometLogger will be initialized in {self.mode} mode")
```

Comment:
Mode has a different meaning now with `comet_ml.start`. The `mode` attribute is not currently documented https://lightning.ai/docs/pytorch/stable/extensions/generated/lightning.pytorch.loggers.CometLogger.html#lightning.pytorch.loggers.CometLogger so I'm fine hiding it under `self._online` and `self._mode` if it's easier to make them match with `comet_ml.start`